Sources/Core/AWSClientRuntime/Errors/UnknownAWSHTTPServiceError.swift

```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
+import class SmithyHTTPAPI.HttpResponse
 import ClientRuntime
```

Comment:
The following files (the event stream files in `AWSClientRuntime`) have been moved to smithy-swift since they are not actually AWS-specific.